components/content/inspira/ui/photo-gallery/PhotoGallery.vue

```diff
@@ -0,0 +1,69 @@
+<template>
+  <div
+    class="gallery"
+    :class="cn('mb-[var(--size)] grid grid-cols-6 gap-1', props.containerClass)"
+  >
+    <img
+      v-for="(image, index) in props.items"
+      :key="index"
+      :src="image.src"
+      :alt="`image+${index}`"
+      class="gallery-img"
+      :class="
+        cn(
+          'size-[calc(var(--size)*2)] rounded object-cover transition-[clip-path,filter] duration-75',
+          props.class,
+        )
+      "
+    />
+  </div>
+</template>
+
+<script setup lang="ts">
+import { cn } from "~/lib/utils";
+
+interface Props {
+  containerClass?: string;
+  class?: string;
+  items: {
```

Comment:
Is it necessary to add a sub object to items? If it has only `src`, then I think `items` can be changed to `images: string[]`.
Can you clarify on that.

Reply:
In future If we want to add more fields related to each `item` for example `title`, no need to change the structure, so for future support added as Array of objects